src/server.js

```diff
@@ -314,19 +314,26 @@ const PATH_NAME = process.env.NODE_ENV === "production" ?
   '~/gazelle-production/scripts/restartServers.sh' : process.env.NODE_ENV === "beta" ?
   '~/gazelle-beta/scripts/restartServers.sh' : __dirname + '/scripts/restartServers.sh';
 
+var isrestarted = false;
```

Comment:
Also, this should be a `let`, we generally never use var anymore as it has a less predictable scope